tests/tkupchyn/homework_03/test_consecutive_strings.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+from tkupchyn.homework_03.consecutive_strings import consecutive_concatenation
+
+
+@pytest.mark.parametrize("list_of_strings, k, expected_result",
+                         [
+                             (["tree", "foling", "trashy", "blue", "abcdef", "uvwxyz",
+                               "hehhnns", "concatenate", "strings", "function", "Python"], 3,
+                              "concatenatestringsfunction"),
+
+                             (["interface", "is", "Up", "issue", "is", "consistent"], 2, "isconsistent"),
+
+                             (["interface", "connectivity", "Up", "issue", "is", "consistent"], 3,
+                              "interfaceconnectivityUp"),
+
+                             (["After", "configuring", "failover"], 1, "configuring"),
+                         ])
```

Comment:
choose codestyle for Parametrize and stick to it, since now you have 2 kinds how you ident parametrize data